### PR TITLE
Add a line in the scan result of the bacterial vat to see its current efficiency

### LINKED ISSUE #9341: add a line in the scan result of the bacterial vat to see its current efficiency
### Your GTNH Discord Username

boubou_19#2706

### Your Pack Version

2.1.2.0

### Your Proposal

add a line in the scan result of the bacterial vat to see its current efficiency.

### Your Goal

it's quite tedious to look at the bacterial vat code for the efficiency and then apply it to your output hatch capacity to see what level of fluid corresponds to which effenciency. There is a rumor saying that the bacterial vat needs to be exactly at 50% of its output hatch capacity to reach maximum while in fact it needs to be between 48.42% and 51.58%. But for that i had to look at the code, and make my own simulation.

### Your Vision

This would help players discovering at what efficiency their bacterial vat is running, allowing for easier management of the bacterial vat.

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying *I like this feature please reconsider adding it* will prompt us to investigate and reopen it once we confirm your report.
- [X] I believe there is nothing similar in the pack already, or the existing solution isn't good enough.
- [X] I understand this change request may not attract enough attention and thus not be implemented.
- [X] I understand this change request may be rejected due to other community members thinking it's inappropriate.
- [X] I believe this feature would make the pack better.

### PR DESCRIPTION
**Pull request by MuXiu1997**
Closed https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/9341

### CODE REVIEW

**Comment by LewisSaber** [comment]
long-ass-github-branch-name

**Comment by MuXiu1997** [comment]
Lazy direct use of the title of the issue

**Comment by LewisSaber** [comment]
At least u dont name material longasssuperconduit

**Comment by MuXiu1997** [comment]
75990CA9A3853BD3532E44B689D24675

File: src/main/java/com/github/bartimaeusnek/bartworks/common/tileentities/multis/GT_TileEntity_BioVat.java [hunk 1]
```diff
@@ -276,80 +308,67 @@ private List<FluidStack> getFluidInputs(){
...
-            GT_Recipe gtRecipe = gtRecipeMap.findRecipe(this.getBaseMetaTileEntity(), this.mLastRecipe, false, this.getMaxInputVoltage(), tFluids, itemStack, tInputs);
+        GT_Recipe gtRecipe = gtRecipeMap.findRecipe(this.getBaseMetaTileEntity(), this.mLastRecipe, false, this.getMaxInputVoltage(), tFluids, itemStack, tInputs);
 
-            if (gtRecipe == null)
-                return false;
+        if (gtRecipe == null) return false;
 
-            if (!BW_Util.areStacksEqualOrNull((ItemStack) gtRecipe.mSpecialItems, itemStack))
-                return false;
+        assert gtRecipe.mFluidInputs.length == 1;
```
Comment (Glease) [inline src/main/java/com/github/bartimaeusnek/bartworks/common/tileentities/multis/GT_TileEntity_BioVat.java:322]: Pretty sure there are many biovat recipes with multiple fluid inputs.

Comment (MuXiu1997) [reply src/main/java/com/github/bartimaeusnek/bartworks/common/tileentities/multis/GT_TileEntity_BioVat.java:322]: 1. The biovat recipes in the NEI has only one fluid input slot
2. If the biovat recipe really has multiple fluid inputs, the original code must not work properly

Comment (Glease) [reply src/main/java/com/github/bartimaeusnek/bartworks/common/tileentities/multis/GT_TileEntity_BioVat.java:322]: Sorry, bad recollection 😅 must be thinking about something else.

**Comment by Glease** [comment]
I recall I saw something wrong with this code beyond the fluid input count issue, but I forget the exact details after returning from meal... I'll have a look the other day in game and see if I can recall what went wrong.

**Review by Glease — APPROVED** [review]

**Pull request merged**